pkg/psmdb/mongos.go

```diff
@@ -408,12 +408,13 @@ func MongosService(cr *api.PerconaServerMongoDB, name string) corev1.Service {
 		svc.Labels = mongosLabels(cr)
 	}
 
-	if cr.CompareVersion("1.12.0") >= 0 {
-		svc.Labels = mongosLabels(cr)
-	}
-
 	if cr.Spec.Sharding.Mongos != nil {
 		svc.Annotations = cr.Spec.Sharding.Mongos.Expose.ServiceAnnotations
```

Comment:
We need to have test for it. Please add this case for one of existing test

Reply:
https://github.com/percona/percona-server-mongodb-operator/pull/969/commits/d9e20c5a072d6ae8e5785b9f574604f6a9ccd347